frontend/src/pages/org/archived-item-qa/ui/resources.ts

```diff
@@ -92,20 +92,18 @@ function renderDiff(
 }
 
 export function renderResources(crawlData: ReplayData, qaData: ReplayData) {
-  const noData = html`<div
-    class="m-4 flex flex-col items-center justify-center gap-2 text-xs text-neutral-500"
-  >
-    <sl-icon name="slash-circle"></sl-icon>
-    ${msg("Resources data not available")}
-  </div>`;
+  // const noData = html`<div
```

Comment:
Assuming resources will always exist for now and no data is indicative of an error.